std/process.d

```diff
@@ -133,20 +132,25 @@ version (Posix)
 {
     version (OSX)
     {
-        extern(C) char*** _NSGetEnviron() nothrow;
-        private __gshared const(char**)* environPtr;
-        extern(C) void std_process_shared_static_this() { environPtr = _NSGetEnviron(); }
-        const(char**) environ() @property @trusted nothrow { return *environPtr; }
+        private extern(C) char*** _NSGetEnviron() nothrow;
+        private const(char**) getEnvironPtr() @property @trusted
+        {
+            return *_NSGetEnviron;
+        }
     }
     else
     {
         // Made available by the C runtime:
-        extern(C) extern __gshared const char** environ;
+        private extern(C) extern __gshared const char** environ;
+        private const(char**) getEnvironPtr() @property @trusted
```

Comment:
BTW, as all this is in a `private { }` block, the `private` annotations here are redundant.